frontend/src/citizen-frontend/messages/ThreadView.tsx

```diff
@@ -43,6 +43,14 @@ const TitleRow = styled.div`
     margin-top: ${defaultMargins.L};
   }
 `
+const StickyTitleRow = styled(TitleRow)`
+  position: sticky;
+  top: 0;
+  padding: 15px;
+  background: white;
+  height: 100px;
```

Comment:
Use a max-height instead of a fixed height if you want to make sure that the title row doesn't take up all the space. Also, you need to give them to the `<h2>` element.

Reply:
Thanks, will do

Reply:
Yeah, maybe. I don't know how likely it is that the title is super long, but very long titles can block the whole view since the header is now sticky.

Reply:
Hmm yeah, how about a scrollable title box with a max-height? Too wild?

Reply:
Yeah, that's what I tried to suggest in my first comment 😄

Reply:
Oops sorry... need to work on my reading comprehension I guess 😅